libs/blocks/tag-selector/tag-selector.js

```diff
@@ -0,0 +1,123 @@
+import { html, render, useState, useEffect } from '../../deps/htm-preact.js';
+import Picker from '../../ui/controls/TagSelectPicker.js';
+import { loadCaasTags } from '../caas/utils.js';
+
+const TagPreview = ({ selectedTags = [] }) => {
+  const [copyText, setCopyText] = useState('Copy');
+
+  /* c8 ignore next 16 */
+  const getTagString = selectedTags.reduce((rdx, tag, i, arr) => {
+    if (i === arr.length - 1) {
+      rdx += tag;
+    } else {
+      rdx += `${tag},`;
+    }
+    return rdx;
+  }, '');
+
+  const handleClick = () => {
+    navigator.clipboard?.writeText(getTagString);
```

Comment:
You should us a catch.
```suggestion
    navigator.clipboard?.writeText(getTagString)
    .then(() => {
        setCopyText('Copied!');
    })
    .catch(() => {
        setCopyText('Failed to Copy!');
    });
```

Reply:
Updated